dev/tasks/linux-packages/apt/descriptor.json

```diff
@@ -0,0 +1,55 @@
+{
+    "package": {
+        "name": "APT",
+        "repo": "apache-arrow-apt",
+        "subject": "kou",
+        "licenses": ["Apache-2.0"],
+        "vcs_url": "htttps://github.com/apache/arrow.git"
+    },
+    "version": {
+        "name": "0.9.0"
+    },
+    "files": [
+        {
+            "includePattern": "cpp-linux/apt/repositories/([^/]+)/pool/stretch/main/a/apache-arrow/([^/]+\\.deb)\\z",
```

Comment:
We may need to change the path. `cpp-linux/apt/` is a path in the `arrow-dist` repository.